app/react/V2/Components/Forms/InputField.tsx

```diff
@@ -60,9 +60,11 @@ const InputField = React.forwardRef(
 
     return (
       <div className={className}>
-        <Label htmlFor={id} hideLabel={hideLabel} hasErrors={Boolean(hasErrors || errorMessage)}>
-          {label}
-        </Label>
+        {label && (
+          <Label htmlFor={id} hideLabel={hideLabel} hasErrors={Boolean(hasErrors || errorMessage)}>
```

Comment:
Why is there a conditional to show the label if the component already has a prop to hide it?

Reply:
Fixed